src/js/create.js

```diff
@@ -0,0 +1,105 @@
+function headerComponent() {
+  const header = document.createElement('header');
+  // refresh button
+  const button = document.createElement('button');
+  button.classList.add('btn', 'refresh');
+  button.innerHTML = '<i>&#x21bb;</i>';
+  header.appendChild(button);
```

Comment:
Instead of separating with the comments. 
U could have just wrap button logic into simple function `createButton` and have such implementation.
```
function createButton() {
const button = document.createElement('button')
button.id = "child"
return button
}
```
and have such implementation within header component:
```
const header= document.getElementById('header');
father.append(createButton())
```

It is a clean code rule - function have to be dedicated to only one idea/purpose. So you could have a nice small function for button (also name would be self-explanatory) so you would not have to provide additional comments and it would be easier to read.


Ps. Love the `const` usage :)

Reply:
agree, done.